_includes/v21.1/backups/backup-options.md

```diff
@@ -2,6 +2,6 @@
 -----------------------------------------------------------------+-------------------------+------------------------------
```

Comment:
Overall, this looks good! I like the phrasing. Can you also update the example on the `BACKUP` pages with the same phrasing to take out "immediately"? https://www.cockroachlabs.com/docs/v22.1/backup#run-a-backup-asynchronously.